frameworks/Rust/hyper/setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sed -i 's|tcp(.*:3306)|tcp('"${DBHOST}"':3306)|g' src/main.rs
+
+cargo build --release
```

Comment:
Cargo would not be on your path at this point - try pointing to it directly `$IROOT/rust/bin/cargo` or whatever the sandboxed path will be.